packages/runtime-node/src/node-environment.ts

```diff
@@ -47,22 +47,9 @@ export async function runNodeEnvironment<ENV extends AnyEnvironment>({
         type,
         env,
     });
-    const rootFeatureLoader = featureLoaders[featureName];
-    if (!rootFeatureLoader) {
-        throw new Error(
-            "cannot find feature '" + featureName + "'. available features: " + Object.keys(featureLoaders).join(', ')
-        );
-    }
-    const { resolvedContexts = {} } = rootFeatureLoader;
-
-    const featureLoader = new FeatureLoadersRegistry(new Map(Object.entries(featureLoaders)), resolvedContexts);
-    const optionsRecord: Record<string, string | boolean> = {};
 
-    for (const [key, val] of options || []) {
-        optionsRecord[key] = val;
-    }
-    const loadedFeatures = await featureLoader.getLoadedFeatures(featureName, optionsRecord);
-    const runningFeatures = [loadedFeatures[loadedFeatures.length - 1]!];
+    const featureLoader = new FeatureLoadersRegistry(new Map(Object.entries(featureLoaders)));
+    const { entryFeature, resolvedContexts } = await featureLoader.loadEntryFeature(featureName, toRecord(options));
```

Comment:
why not use `Object.fromEntries(options || [])` ?

Reply:
done